web/app/themes/justicejobs/src/js/accessible-carousel.js

```diff
@@ -3,271 +3,278 @@
+    var myCarousel = (function () {
 
-    "use strict";
+        "use strict";
 
-    // Initial variables
-    var carousel, slides, index, slidenav, settings, timer, setFocus, animationSuspended, announceItem, _this;
+        // Initial variables
+        var carousel, slides, index, slidenav, settings, timer, setFocus, animationSuspended, announceItem, _this;
 
-    // Helper function: Iterates over an array of elements
-    function forEachElement(elements, fn) {
-      for (var i = 0; i < elements.length; i++)
-        fn(elements[i], i);
-    }
+        // Helper function: Iterates over an array of elements
+        function forEachElement(elements, fn) {
+            for (var i = 0; i < elements.length; i++)
+                fn(elements[i], i);
+        }
 
-    // Helper function: Remove Class
-    function removeClass(el, className) {
-      if (el.classList) {
-        el.classList.remove(className);
-      } else {
-        el.className = el.className.replace(new RegExp('(^|\\b)' + className.split(' ').join('|') + '(\\b|$)', 'gi'), ' ');
-      }
-    }
+        // Helper function: Remove Class
+        function removeClass(el, className) {
+            if (el.classList) {
+                el.classList.remove(className);
+            } else {
+                el.className = el.className.replace(new RegExp('(^|\\b)' + className.split(' ').join('|') + '(\\b|$)', 'gi'), ' ');
+            }
+        }
 
-    // Helper function: Test if element has a specific class
-    function hasClass(el, className) {
-      if (el.classList) {
-        return el.classList.contains(className);
-      } else {
-        return new RegExp('(^| )' + className + '( |$)', 'gi').test(el.className);
-      }
-    }
+        // Helper function: Test if element has a specific class
+        function hasClass(el, className) {
+            if (el.classList) {
+                return el.classList.contains(className);
+            } else {
+                return new RegExp('(^| )' + className + '( |$)', 'gi').test(el.className);
+            }
+        }
 
-    // Initialization for the carousel
-    // Argument: set = an object of settings
-    // Possible settings:
-    // id <string> ID of the carousel wrapper element (required).
-    // slidenav <bool> If true, a list of slides is shown.
-    function init(set) {
+        // Initialization for the carousel
+        // Argument: set = an object of settings
+        // Possible settings:
+        // id <string> ID of the carousel wrapper element (required).
+        // slidenav <bool> If true, a list of slides is shown.
+        function init(set) {
+            // Make settings available to all functions
+            settings = set;
+
+            // Select the element and the individual slides
+            carousel = document.getElementById(settings.id);
+
+            // if no carousel available, bail
+            if (!carousel) {
+                return false;
+            }
 
```

Comment:
Above is the addition I made - lines 83 - 86. The rest is auto-formatting by my IDE, sorry about that!